models/tokens/avalanche_c/tokens_avalanche_c_erc20.sql

```diff
@@ -1,221 +1,217 @@
+        ,(0x0db995f93dd05ae966ab4c57e43b443a9b18a532,'MASK',18)
+        ,(0x2b2c81e08f1af8835a78bb2a90ae924ace0ea4be,'sAVAX',18)
+        ,(0x5817d4f0b62a59b17f75207da1848c2ce75e7af4,'VTX',18)
+        ,(0x5541d83efad1f281571b343977648b75d95cdac2,'GRAPE',18)
+        ,(0x4e51e67f1c6da0f9ba52fe27a6e9a9dbd36ac479,'FDT',18)
+        ,(0x73cf93261a5993415cab9ce07bfd8411e3f361cc,'X',18)
+        ,(0x54a77f27d2346c3a8f5b43a501434f0f21f33e3c,'KGC',18)
+        ,(0x9366d30feba284e62900f6295bc28c9906f33172,'BioFi',6)
+        ,(0x31c994ac062c1970c086260bc61babb708643fac,'XETA',18)
+        ,(0x093783055f9047c2bff99c4e414501f8a147bc69,'ALOT',18)
+        ,(0x77777777777d4554c39223c354a05825b2e8faa3,'YETI',18)
+        ,(0x3eefb18003d033661f84e48360ebecd181a84709,'ISA',18)
+        ,(0x59b18817ca9f4ad2dee6fbf30132df6aeb9d763d,'AVIC',18)
+        ,(0xa577d9bff8846bba0e126388bf26252683695a50,'WAVAX',18)
+        ,(0x08f9492ee68594fcc356f9d91af93d8e4f8c9f33,'ETHW',18)
+        ,(0x5a15bdcf9a3a8e799fa4381e666466a516f2d9c8,'SLIME',18)
+        ,(0x9a8e0217cd870783c3f2317985c57bf570969153,'MAGIC',18)
+        ,(0xb6c353d519d7721b18c813130625d04de4f53580,'BRISE',18)
+        ,(0x100cc3a819dd3e8573fd2e46d1e66ee866068f30,'DCAU',18)
+        ,(0x7b2b702706d9b361dfe3f00bd138c0cfda7fb2cf,'PLN',18)
+        ,(0x70928e5b188def72817b7775f0bf6325968e563b,'LUNA',6)
+        ,(0x9e15f045e44ea5a80e7fbc193a35287712cc5569,'3ULL',18)
+        ,(0x5c49b268c9841aff1cc3b0a418ff5c3442ee3f3b,'MAI',18)
+        ,(0x3712871408a829C5cd4e86DA1f4CE727eFCD28F6,'GLCR',18)
+        ,(0x2dc3Bb328000553D1D64ec1BEF00572F62B5Ec7C,'XWLRS',18)
+        ,(0xfDECfC325B584A96E86caBD2eb492320467ed1d8,'MIGHT',18)
+        ,(0xF7554D17d1c3F09899dCc8B404beCAE6dFA584Fa,'MAGIK',18)
+        ,(0x0da67235dD5787D67955420C84ca1cEcd4E5Bb3b,'WMEMO',18)
+        ,(0xa41A879bcFdd75983a987FD6b68fae37777e8b28,'XEX',18)
+        ,(0x5c09a9ce08c4b332ef1cc5f7cadb1158c32767ce,'fBOMB',18)
+        ,(0x2c69095d81305F1e3c6ed372336D407231624CEa,'OATH',18)
+        ,(0xE98786B94520772DD9efcC670d72939e184A6198,'spGLCR',18)
+        ,(0xA0105D7fc6190598523f568001A71164341b0A22,'SUB',18)
+        ,(0xDE1E704dae0B4051e80DAbB26ab6ad6c12262DA0,'DEI',18)
+        ,(0xF7D9281e8e363584973F946201b82ba72C965D27,'yyAVAX',18)
+        ,(0x9df4Ac62F9E435DbCD85E06c990a7f0ea32739a9,'GRAIN',18)
+        ,(0x1e3c6c53F9f60BF8aAe0D7774C21Fa6b1afddC57,'xSHRAP',18)
+        ,(0x089d3dAF549f99553C2182DB24bC4336A4F0C824,'IBEX',18)
+        ,(0xb36FAF341c7817D681F23BceDbd3d85467E5aD9F,'RPEPE',18)
+        ,(0xC55036B5348CfB45a932481744645985010d3A44,'WINE',18)
+        ,(0x01Af64EF39AEB5612202AA07B3A3829f20c395fd,'VINTAGE',18)
+        ,(0x501012893eE88976AB8B5289B7a78E307d5d9DCb,'NRWL',18)
+        ,(0x09EF821c35B4577f856cA416377Bd2ddDBD3d0C9,'MMTH',8)
+        ,(0x1C1CDF8928824dac36d84B3486D598B9799bA6c0,'aBASED',18)
+        ,(0xAEC8318a9a59bAEb39861d10ff6C7f7bf1F96C57,'agEUR',18)
+        ,(0x8901cB2e82CC95c01e42206F8d1F417FE53e7Af0,'YFX',18)
```

Comment:
why are 3-4 below removed here? i notice they are included in the legacy file, but removed here?